pkgs/development/libraries/boxfort/default.nix

```diff
@@ -1,14 +1,14 @@
 { lib, stdenv, fetchFromGitHub, meson, ninja, python3Packages }:
 
 stdenv.mkDerivation rec {
-  version = "unstable-2019-10-09";
+  version = "unstable-2022-02-19";
   pname = "boxfort";
```

Comment:
```suggestion
  pname = "boxfort";
  version = "0.1.4";
```

Reply:
Oh, boxfort got version numbers? I didn't notice.

Reply:
There is a tag on that commit.